en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
     
-2. Sign in using either your Google or GitHub credentials.
-    
-3. In the **Services** page, click **Create**. You are directed to the low-code view.
+2. On the **Services** page, click **Create**. You are directed to the **Create Service** page.
 
-4. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
+3. Under **Create with Choreo**, enter `covid-stats` as the name of your service.
 
-5. Select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
+4. Select **GET** as the HTTP method and enter `activecases` as the relative path from the host.
 
-    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
-    
-    Then click **Save API**. Now you have saved the API trigger that starts your service.
-    
-6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+     ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
+
+5. Click **Save API**. Now you have saved the API trigger that starts your service.
+
+6. Click **API Calls** and then complete the following steps to configure the API trigger to use the HTTP connection that connects the public API used in this scenario to it:
 
     1. Click **HTTP**.
     
         ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
-        
-    2. Enter information as follows:
-    
-        | **Field**           | **Value**                    |
-        |---------------------|------------------------------|
-        | **Connection Name** | `httpEndpoint`               |
-        | **URL**             | `"https://api.covid19api.com"` |
-        
-        Click **Save**.
-        
+         
+    2. Enter the following information and then click **Save**:
+    
+        | **Field**           | **Value**                      |
+        |---------------------|--------------------------------|
+        | **Connection Name** | `httpEndpoint`                 |
+        | **URL**             | `https://api.covid19api.com`   |
+         
     3. Click the **+** icon below the HTTP API call you added. 
     
         ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
         
-    4. Click **API Calls**, and then click on the existing connection.
+    4. Click **API Calls** and then click the existing connection.
```

Comment:
We can refer to the existing connection by the connection name we added above, can't we?

Reply:
Since the UI shows the added connection as an existing connection, the current sentence seems ok. 
<img width="894" alt="Screenshot 2021-05-28 at 10 12 49" src="https://user-images.githubusercontent.com/9528121/119931028-3b909080-bf9e-11eb-8cb1-56f980d2a9bd.png">

I feel the sentence might become longer if we try to mention the name. WDYT?